dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -825,6 +832,24 @@ private GHRateLimit getGhRateLimitQuietly() {
         return startRateLimit;
     }
 
+    private GHRef[] getBranchesAndTags(GHRepository repo) {
+        GHRef[] branches = {};
+        GHRef[] tags = {};
+
+        try {
+            branches = repo.getRefs("refs/heads/");
+        } catch (IOException ex) {
```

Comment:
Hmm, not sure about this and the other catch below, at least for the hotfix. Things that would throw errors before will now either fail silently or in a different way. I think the method should throw an IOException. Or is there a reason it's needed because the behavior of the two calls is different?